Workbooks/Workloads/Onboard MySQL Insights/Onboard MySQL Insights.workbook

```diff
@@ -29,7 +29,7 @@
             "name": "Workspace",
             "type": 5,
             "isRequired": true,
-            "query": "Resources\r\n| where type =~ 'microsoft.compute/virtualmachines/extensions' and name =~ 'OmsAgentForLinux' and id startswith '{VirtualMachine}'\r\n| project WorkspaceId = tostring(properties.settings.workspaceId)\r\n| join kind = inner (Resources\r\n| where type =~ 'microsoft.operationalinsights/workspaces'\r\n| project id, WorkspaceId = tostring(properties.customerId)) on WorkspaceId\r\n| project value = id, label = id, selected = true",
+            "query": "Resources\r\n| where type =~ 'microsoft.compute/virtualmachines/extensions' and name in~ ('OmsAgentForLinux', 'OMSExtension') and id startswith '{VirtualMachine}'\r\n| project WorkspaceId = tostring(properties.settings.workspaceId)\r\n| join kind = inner (Resources\r\n| where type =~ 'microsoft.operationalinsights/workspaces'\r\n| project id, WorkspaceId = tostring(properties.customerId)) on WorkspaceId\r\n| project value = id, label = id, selected = true",

```

Comment:
properties.type == 'OmsAgentForLinux' and properties.publisher == 'Microsoft.EnterpriseCloud.Monitoring'

Reply:
Doug, I am assigning you will make this change across all templates including this one.